code/modules/mob/living/say.dm

```diff
@@ -232,6 +232,17 @@ proc/get_radio_key_from_channel(var/channel)
 	var/italics = 0
 	var/message_range = world.view
 
+	// Diona without head can live, but they cannot talk as loud anymore.
+	if(src.is_diona())
+		var/mob/living/carbon/human/h = src
+		var/obj/item/organ/O = h.organs_by_name["head"]
+		if(istype(O, /obj/item/organ/external/stump))
+			message_range = 3
+			sound_vol *= 0.5 //muffle speech
+			italics = 1
+			visible_message("<span class='notice'>[src] says something very quietly.</span>")

```

Comment:
I think this would lead to unnecessary spam if not limited to the people who cannot hear it. In gameplay it will look like:

Dionaea says, "Dab on the haters."
Dionaea says something very quietly
Dionaea says, "Fuck bernie sanders."
Dionaea says something very quietly.

Despite being in range.

Reply:
You are right, I will rework to include in the loops below